types/__tests__/type-tests.ts

```diff
@@ -178,15 +178,9 @@ export async function testByRole() {
   console.assert(queryByRole(element, 'button', {name: /^Log/}) === null)
   console.assert(
     queryByRole(element, 'button', {
-      name: (name, el) => name === 'Login' && el.hasAttribute('disabled'),
+      name: (name, el) => name === 'Login' && !!el?.hasAttribute('disabled'),
```

Comment:
why does this need to change?

It's also pretty hard to grok what this is doing.

Reply:
Name require a boolean value. `el` variable could be undefined based on typing. The boolean casting is because type result could be udefined by `el` or boolean by `hasAttribute`

Maybe is more readable like that

```ts
name: (name, el) => name === 'Login' && !!el && el.hasAttribute('disabled'),
```

Reply:
Why is `el` nullable? Shouldn't that always be defined?